routes/api.php

```diff
@@ -14,8 +13,9 @@
 |
 */
 
-Route::middleware('auth:api')->get('/user', function (Request $request) {
-    return $request->user();
-});
+Route::post('register', 'AuthController@register')->name('register');
+Route::post('login', 'AuthController@login')->name('login');
+Route::post('logout', 'AuthController@logout')->name('logout')->middleware('auth:sanctum');
+Route::get('user', 'AuthController@user')->name('getUser')->middleware('auth:sanctum');
```

Comment:
Please move /user route under the auth middleware as the user must be authenticated to retrieve profile.

Reply:
Route::get('user', 'AuthController@user')->name('getUser')->**middleware('auth:sanctum')**;

/user already has auth middleware